Content: Remnant culture: Paradise 1 - Historic Spaceport (#6655)
endless-sky/endless-sky: data/remnant/remnant 2 side missions.txt

			choice
				`	"Yes, that sounds fun."`
					goto vailfun
				`	"Sure, I have done a bit of hoverboarding myself."`
				`	"Maybe later."`
					goto vaillater
			apply
				set "remnant: hoverboarding"
			`	"Really? The description sounded useful for exploration, and it looked challenging." Her commlink shifts to display a hologram of a hoverboarder executing a jump with an artistic poof of snow around them. "Anyway, I think this trip should be a good diversion for us both. I will meet you onboard the <ship> shortly."`
				accept
			label vailfun
			`	"Excellent. I will meet you onboard the <ship> shortly."`
				accept
	on defer
		set "Remnant: Visit to Paradise 1: defer"
	on stopover
		conversation
			branch science
				has "Research Laboratory"
			`As you approach, your navigation system picks up the beacons designating the approach path, and before long you are automatically guided to a landing pad on the outskirts of the main spaceport.`
				goto vaillanding
			label science

> 			`As you approach Vail, your sensors detect parts of the weather control network tasked with increasing the amount of snowfall on the planet. Your navigation system picks up the beacons designating the approach path, and before too long you are automatically guided down to a landing pad on the outskirts of the main spaceport.`

Reviewer: ```suggestion
			`As you approach Vail, the specialized sensors from your Remnant research laboratory detect parts of the weather control network tasked with increasing the amount of snowfall on the planet. Your navigation system picks up the beacons designating the approach path, and before too long you are automatically guided down to a landing pad on the outskirts of the main spaceport.`
```
Or something like this? Could be nice to note that the reason we're getting this info is the lab.

			label vailheadhome
			`	Heading back to the spaceport after a day of hoverboarding, Taely surprises you by pulling the hood open on the hovercar and spending almost an hour going over the internal systems before closing it up with a nod. "Well, that was interesting. The hovercar, I mean." She gestures at it with her commlink. "So many different manufacturers were involved in making this vehicle. Sometimes I wonder if humanity appreciates the luxury such diversity affords them." She looks thoughtful, then gestures toward the ship. "Well, <first>, I promised you a tour of a historic Remnant district. Caelian has perhaps the best example of one, so I would recommend we go there."`
	on complete
		conversation
			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with Gulls and Pelicans shifting larger quantities of products. Disembaring from the <ship>, she leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a few minutes they deposit you on a large platform.`

> 			`	Approaching the edge of the platform, Taely puts out her arm to stop you from advancing any farther. "Watch out for the barrier," she says as she puts a hand out and touches something in mid-air in front of you. As she leans on it you can see faint ripples around her hand. "They are an essential safety feature, but they can still hurt if you run into them." She gestures to easily recognizable hazard markings on the floor. "With time you will learn to notice the faint shield ripples in the air, but meanwhile, just remember that anywhere that has the 'stand-back' lines on the floor likely has shields above it most of the time."`

Reviewer: All I can think about is a scenario where firstlast walks straight into one of these and smacks right into it, full glass-door style :P

Author: Maybe I will have that happen in a different side mission, one where their Remnant escort is distracted or something.

> 			`	Moments later, you see why the barriers are in place as a large train whooshes into the station at high speed before quickly coming to a stop. The barrier vanishes, and Taely leads you onto the train. Looking up and down the length of it, you can see hundreds, maybe even thousands of Remnant entering and exiting the train. Within moments the doors close and the train starts moving again as you are pressed back into your seat. You see a few lights flicker past outside the window, but soon there is nothing outside but a monotonous dark blur.`

Reviewer: ```suggestion
			`	Moments later, you see why the barriers are in place as a large train whooshes into the station at high speed before quickly coming to a stop. The barrier vanishes, and Taely leads you onto the train. Looking up and down the length of it, you can see hundreds, maybe even thousands of Remnant entering and exiting the train. Within moments the doors close, and the train starts moving again as you are pressed back into your seat. You see a few lights flicker past outside the window, but soon there is nothing outside but a monotonous dark blur.`
```

			`	"The C3-L0 Maglev has a maximum rated velocity of 11D.E8 meters per second, and can sustain that almost indefinitely so long as the track remains powered. In practice, it rarely maintains that velocity for more than a few minutes at a time, as it is a local transport train. We do have some that go faster, but the amount of acceleration required to make use of them on short tracks is not terribly practical." She looks wistful. "Many people are working on solving the inertia problem, but Embers only know when that will result in something practical."`
			choice
				`	"Interesting. How long until we arrive?"`
					goto ourdestination
				`	"That number seems familiar."`
				`	"That sounds like the speed of light."`
			`	Taely smiles. "After all the safety factors, margins for error, and assorted other adjustments were taken into account, someone noticed that the rated value was a fraction of a meter per second above the coefficient of c, so they adjusted it down slightly so that the rated maximum velocity would match the coefficient of the speed of light in a vacuum.`
			choice
				`	"Thanks for the explanation."`
					goto ourdestination
				`	"What is a coefficient?"`

> 			`	She pulls up a hologram with c=11'DE7'84Am/s on it. "Here, I will switch to base 10 to make it easier for you to understand." As she speaks, the number 299'792'458m/s appears on the screen, and then changes to 2.997'924'58x10^8m/s. "When we write out a number such as the speed of light, it is typical to break it into parts. This is generally referred to as scientific notation. This is done to facilitate comparing and working with large numbers. The first part, 2.997'924'58 in this case, would be called the coefficient. The second part, x10, specifies the base; and the third part, ^8 is the exponent." The display shifts again to show 'c=1.1DE784Ax16^7m/s'. "So this would be how we typically write the speed of light when we are using it in our research." She pauses, then vanishes the display. "We will be arriving momentarily, so I should probably stop there."`

Reviewer: Is the use of apostrophes to separate digits a hexadecimal thing, Remnant thing, or neither? Some of your notation seems odd here - I can't understand it, and it isn't explained. For context, American English would use commas to separate every three whole digits and periods to separate between whole numbers and decimals. For instance, one thousand five hundred and four hundredths would be "1,500.04" (I promise I'm not trying to be patronizing if it comes off this way, just trying to understand what the intent here is and make sure what I'm thinking is clear).

Author: Let's call it a Remnant thing. It's actually a personal habit, because I've had to deal with a lot of places where the , and . and things look too similar and generate confusion. Given that the Remnant are kind of a high-pressure "need to understand stuff at a glance" types, it makes sense that they'd adopt a digit grouping separator that won't generate confusion and erroneous readings.

Reviewer: Got it. I don't know if it's just me, but my concern here is that your numbering system is abnormal enough that your explanation that uses it loses its usefulness as an explanation by using something that's not well-known (at least in my experience). For instance, I have no idea what number 2.997'924'58 actually is.

Author: I see your point. I think it probably works to the left of the decimal, but it gets really hard to read in the scientific notation. I think that is a case of one or the other, not both.

Author: Alright. This has been changed. Added a bunch of spaces to spread it out a bit; and took all the apostrophes out of the scientific notation.

			label ourdestination
			`	After several minutes, the lights in the train car suddenly pulse several times, and the train starts to decelerate heavily. You can feel yourself pushing into the harness as the train slows and comes to a stop. Unbuckling, Taely leads you out onto the platform, and then up a long escalator to the surface. Emerging into the sunlight, you find yourself looking around at a scene that seems almost anachronistic.`
			scene "scene/remnant remote spaceport"
			`	Taely looks around with pride, and then watches your reaction with amusement.`
			choice
				`	"I do not know what to make of this."`
				`	"This is a spaceport?"`

> 			`	She nods, and guides you amongst the stream of camel-drawn hovercarts heading for the gate. "This is one of our earliest spaceports, before we learned new techniques better suited to our new home. Our founding ships brought as much as they could, of course, but as it turns out, the earliest phases of expansion look much the same whether one is looking at historic pre-flight eras, or modern settlement efforts."`

Reviewer: ```suggestion
			`	She nods, and guides you amongst the stream of camel-drawn hovercarts heading for the gate. "This is one of our earliest spaceports, before we learned new techniques better suited to our new home. Our founding ships brought as much as they could, of course, but as it turns out, the earliest phases of expansion look much the same, whether one is looking at historic pre-flight eras or modern settlement efforts."`
```

			scene "scene/remnant remote spaceport"
			label questionloop
			choice
				`	"It still seems odd to see camels pulling levitating platforms."`
					goto camelcart
				`	"Stone walls seem like an odd thing to build."`
					goto stonewalls
				`	"Are those wind turbines?"`
					goto windturbine

> 				`	"All these wood products seems surprising."`

Reviewer: ```suggestion
				`	"All these wood products seem surprising."`
```

				`	"I think I have seen enough."`
					goto mainreturn
			label woodproducts
			scene "scene/remnant remote spaceport"
			`	Taely picks up a board from a nearby pile. "We were fortunate that Caelian had some vegetation that could be used like this. The native vegetation is biologically quite different from the trees of Earth, but they have similar structural characteristics and grow plentifully in many areas." She offers you the board. "The decision was made early on that anything that could be made from locally sourced materials, should be sourced locally. We wanted to conserve our limited supplies of metals and other rare resources for things that required it." She taps on a barrel. "These are a prime example - a few metal straps enable the wood to hold a significant amount of liquid. It is much the same as how a comparatively small amount of bone in our bodies can provide the necessary structure for a large volume of soft tissues and organs to function."`
				goto questionloop
			label camelcart
			scene "scene/remnant remote spaceport"

> 			`	"Is that so? The engineering advantages are quite simple: With the mechanics of the cart focused entirely on supporting the load and the camel providing horizontal motion, the cart can support much larger masses than it could if it had to split the forces." She looks wistful. "In truth, I am somewhat envious of the industrial capacity the ancestral worlds contain - We have accomplished much, but we have had to make choices, and one of those was the decision as to where to spend our industrial capacity. We chose, obviously, to focus on spacecraft and heavy industrial applications. We have no cars, no personal vehicles at all, really. Even these hovercarts are basically the repulsor systems from our spacecraft that have been repurposed to carry massive loads."`

Reviewer: ```suggestion
			`	"Is that so? The engineering advantages are quite simple: with the mechanics of the cart focused entirely on supporting the load and the camel providing horizontal motion, the cart can support much larger masses than it could if it had to split the forces." She looks wistful. "In truth, I am somewhat envious of the industrial capacity the ancestral worlds contain - we have accomplished much, but we have had to make choices, and one of those was the decision as to where to spend our industrial capacity. We chose, obviously, to focus on spacecraft and heavy industrial applications. We have no cars, no personal vehicles at all, really. Even these hovercarts are basically the repulsor systems from our spacecraft that have been repurposed to carry massive loads."`
```

				goto questionloop
			label stonewalls
			scene "scene/remnant remote spaceport"
			`	Taely's expression becomes mournful as she leads you over to the walls. "We discovered soon after we arrived here that this planet was not as safe as it appeared. It had a variety of natural predators, including swarms of predatory creatures that were very aggressive in hunting us. They preferred darkness, and while they had excellent eyesight, their hearing was even sharper. A swarm could triangulate the position of a person from a single spoken word at great distances."`
			`	She gestures up at the edge of the wall. "Fortunately, we discovered that while they were quite competent climbers, they could not navigate overhangs. So while such lips are generally considered a structural vulnerability when defending against modern weapons, they are indispensable for containing local wildlife. These walls served as well later on, too, when we had to deal with Korath ground attacks and other more complex threats."`
				goto questionloop
			label windturbine
			scene "scene/remnant remote spaceport"

> 			`	"They are indeed wind turbines." She steps through the gate to a spot with a better view. You can see that while the ones on the wall are are only a couple of meters tall, the ones on top of the towers are significantly bigger. "Our ancestors rightly guessed that anywhere that had sufficient atmosphere to dispense with pressure suits would also have wind. The vertical style also has the advantage that it handles high wind speeds and rapid variation in direction much better than some of the shapes that were more traditional at the time of the exodus. A good precaution, which has enabled these to survive the harsh windstorms that can sweep across the continent. While they have been repaired and improved over the centuries, these wind turbines still provide sufficient power for much of the agricultural uses out here, so we let them continue. They are also useful to remind us from where we have come."`

Reviewer: Two concerns:
1. `The vertical style also has the advantage that it handles high wind speeds and rapid variation in direction much better than some of the shapes that were more traditional at the time of the exodus.`
Are we missing a word here? Can't understand the `direction` bit so I thought that might be the case

2. `A good precaution, which has enabled these to survive the harsh windstorms that can sweep across the continent.`
This is an incomplete sentence. I'd suggest a change, but I just need to understand what the first one's saying before I come up with something.

Author: 1. "rapid variation in direction" or in other words, dealing with winds that change directions really frequently and abruptly.

Reviewer: ```suggestion
			`	"They are indeed wind turbines." She steps through the gate to a spot with a better view. You can see that while the ones on the wall are are only a couple of meters tall, the ones on top of the towers are significantly bigger. "Our ancestors rightly guessed that anywhere that had sufficient atmosphere to dispense with pressure suits would also have wind. The vertical style also has the advantage that it handles high wind speeds and rapid variation in direction much better than some of the shapes that were more traditional at the time of the exodus. It has enabled these to survive the harsh windstorms that can sweep across the continent. While they have been repaired and improved over the centuries, these wind turbines still provide sufficient power for much of the agricultural uses out here, so we let them continue. They are also useful to remind us from where we have come."`
```
OR
```suggestion
			`	"They are indeed wind turbines." She steps through the gate to a spot with a better view. You can see that while the ones on the wall are are only a couple of meters tall, the ones on top of the towers are significantly bigger. "Our ancestors rightly guessed that anywhere that had sufficient atmosphere to dispense with pressure suits would also have wind. The vertical style also has the advantage that it handles high wind speeds and rapid variation in direction much better than some of the shapes that were more traditional at the time of the exodus. It seems to be a good precaution, as it has enabled these to survive the harsh windstorms that can sweep across the continent. While they have been repaired and improved over the centuries, these wind turbines still provide sufficient power for much of the agricultural uses out here, so we let them continue. They are also useful to remind us from where we have come."`
```
Thanks! Take your pick.

# Copyright (c) 2022 by Zitchas
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE.  See the GNU General Public License for more details.

mission "Remnant: Visit to Paradise 1"
	name "Take Taely to visit <stopovers>"

> 	description "Take Taely to visit Vail, then return to Caelian."

Author: Do we really need to replace these? As in, is there actually a point or a solid reason for doing so? These destinations are hardcoded. There is zero possibility that they will be anything other than Vail and Caelian.

I am increasingly of the opinion that we should *not* use text replacements for cases where there is no possibility of change. This is simply adding in complexity for the sake of adding in complexity, as best I can tell.

> 	description "Take Taely to visit Vail, then return to Caelian."

Reviewer 2: If the destination is changed for whatever reason in the future, then we don't need to update the text every time. Also text replacements aren't that complicated.

Also useful when changing the source/destination for testing purposes I guess.

> 	description "Take Taely to visit Vail, then return to Caelian."

Author: OK, I'm opting to *not* replace them. Making it easy to identify and work with missions and identify which ones are specific locations and which ones can vary. Definitely takes precedence over the non-likely-to-every-be-useful benefit of being able to change them later.

> 	description "Take Taely to visit Vail, then return to Caelian."

Reviewer: I'm firmly on the side of replacing these, myself. Your point is valid, but it doesn't seem reasonable to break that precedent unless there's enough support from folks generally. If you've got concerns, perhaps we could get some more community feedback or something to reach a resolution?

> 	description "Take Taely to visit Vail, then return to Caelian."

Author: I'm willing to go with the tradition for now (and the change has been applied), but I would liked to see some community consensus on having a bit of flexibility in certain places for the future.

> 	description "Take Taely to visit Vail, then return to Caelian."

Reviewer: Sounds good!

	minor
	source
		government "Remnant"
	destination "Caelian"
	stopover "Vail"
	to offer
		has "event: remnant: cognizance calm"

> 		or

Author: It's not, actually. The random value must be greater than 30 to trigger normally. With a defer, the random value needs to be greater than 60. So that should be something close to half as likely to trigger.

> 			and
> 				random > 30
> 				not "Remnant: Visit to Paradise 1: defer"
> 			and
> 				random > 60
> 				has "Remnant: Visit to Paradise 1: defer"

Reviewer 3: Keep in mind that "Mission: defer" conditions aren't generated automatically.
```suggestion
			and
				random < 70
				not "Remnant: Visit to Paradise 1: defer"
			and
				random < 40
				has "Remnant: Visit to Paradise 1: defer"
```

Author: They're not? We have to manually create those when we use a defer?

Reviewer 3: Yes.

Author: Alright, thanks for pointing that out. I'll add in an "on defer" clause to add in that statement.

			choice
				`	"Exciting as always."`
					goto paradiseexciting
				`	"Taking a break to relax a bit."`
			`	She nods sympathetically. "Things have been a bit hectic lately, what with dimensional pockets and mysterious new aliens. I can understand wanting to slow down a bit. With the launch of two new ships in such a short time, I know I feel like I need to just relax a bit." She sighs "It makes me feel guilty, honestly. I have accomplished so much, and there are so many things to be grateful for. I even have more projects in development to focus on." She looks lost in thought for a moment.`
				goto paradisechoice
			label paradiseexciting
			`	"I can only imagine what it must be like, constantly meeting new people and traveling to new places." She eats mechanically, lost in thought for a moment.`
			label paradisechoice
			`	"Could I ask you something about life back in ancestral space?"`
			choice
				`	"Sure."`
					goto askquestion
				`	"Not right now."`
			label vaillater
			`	"Oh, maybe some other time, then." She looks disappointed, and settles in to eat."`

> 				defer

Author: I personally take the few of "Is there any consequence to delaying this? Might the player not want to do this now but do it later? If the answer to the first is no and the second yes, then it should have a defer.

			choice
				`	"That sounds fun."`
					goto vailfun
				`	"I have done a bit of hoverboarding myself."`
				`	"Maybe later."`
					goto vaillater
			apply
				set "remnant: hoverboarding"
			`	"Really? I noted a mention of that in the descriptions, and it looked challenging." Her commlink shifts to display a hologram of a hoverboarder executing a jump with an artistic poof of snow around them. "Anyway, I think this trip should be a good diversion for us both. I will meet you onboard the <ship> shortly."`
				accept
			label vailfun
			`	"Excellent. I will meet you onboard the <ship> shortly."`
				accept
	on stopover
		conversation
			branch science
				has "Research Laboratory"
			`As you approach your navigation system easily picks up the beacons designating the approach path, and before too long you connect to the automated approach control that shunts you into the appropriate route down to a landing pad on the outskirts of the main spaceport.`
				goto vaillanding
			label science

> 			`As you approach Vail your sensors light up with energy traces that it quickly identifies as part of the weather control network. From what you recall, many areas of the planet have been adjusted to have greater than usual snowfall. As you approach closer your navigation system easily picks up the beacons designating the approach path, and before too long you connect to the automated approach control that shunts you into the appropriate route down to a landing pad on the outskirts of the main spaceport.`

Reviewer 3: ```suggestion
			`As you approach Vail, your sensors detect parts of the weather control network tasked with increasing the amount of snowfall on the planet. Your navigation system picks up the beacons designating the approach path, and before too long you are automatically guided down to a landing pad on the outskirts of the main spaceport.`
```

			label vaillanding
			`	Looking outside the ship, you can see a multitude of tourist-oriented facilities overwhelming the normal spaceport functions. You can pick out a small sign off to the side that indicates professional commodity and banking facilities, but otherwise the area is dominated by businesses offering mountain touring, sports classes, and vehicle rentals. Next to you, Taely alternates between peering out at the signs and reading brochures on a new commlink. "Here, <first>. This one sounds more interested in providing a challenge and less pampering. What do you think?"`
			choice
				`	"Looks good to me."`

> 				`	"Alright, should be good."`

Reviewer 3: "Rather than offer you the illusion of free choice, I will take the liberty of choo-sing for you."

Author: That is one thing I have learned from Mass Effect: Illusion of choice actually can be important: The vast majority of "choices" in Mass Effect have zero impact on the course of events of the game. They do, however, let the player shape their character. Am I the rebellious, crass, and aggressive commander that does stuff in the most gratingly hostile way I can? Am I the easy-going and/or professional soldier that accepts things the way they are and gets along with people? Some mix somewhere between the two? 

Exact same course of events, different story.

Reviewer 3: This isn't the illusion of choice, though. You are offering the exact same choice twice. Like at this point, you might as well chuck out the choice node and just jump straight to the boarding.

Author: Fair, but me, as a player, really wants to be able to say something there.

> 			branch "havehoverboarded"

Reviewer 3: ```suggestion
			branch havehoverboarded
```

				has "remnant: hoverboarding"

> 			`	The resort in question is a more exclusive facility, and thus is located away from the main train lines. You rent a hovercar from one of the rental companies and make the short trek out to the resort. The rest of the day is spent out on the slopes, where both you and Taely eventually develop sufficient skill to graduate from the easy runs up to the medium difficulty. You both fall repeatedly, but Taely's good humor is contagious as she takes the inevitable facefuls of snow good naturedly.`

Reviewer 3: ```suggestion
			`	The resort in question is a more exclusive facility, and is located away from the main train lines. You rent a hovercar from one of the rental companies and make the short trek out to the resort. The rest of the day is spent out on the slopes, where both you and Taely eventually develop sufficient skill to graduate from the easy runs up to the medium difficulty. You both fall repeatedly, but Taely's good humor is contagious as she takes the inevitable facefuls of snow in stride.`
```

				goto vailheadhome
			label havehoverboarded

> 			`	The resort in question is a more exclusive facility, and thus is located away from the main train lines. You rent a hovercar from one of the rental companies and make the short trek out to the resort. The rest of the day is spent out on the slopes, where you teach Taely the basics of hoverboarding. By the end of the day Taely has impressed you with her quick learning and you finish off with a few runs down some of the more challenging slopes. Taely spills a few times, but she takes her failures in good humor, and never seems to make the same mistake twice.`

Reviewer 3: ```suggestion
			`	The resort in question is a more exclusive facility, and is located away from the main train lines. You rent a hovercar from one of the rental companies and make the short trek out to the resort. The rest of the day is spent out on the slopes, where you teach Taely the basics of hoverboarding. By the end of the day Taely has impressed you with her quick learning and you finish off with a few runs down some of the more challenging slopes. Taely spills a few times, but she takes her failures in good humor, and never seems to make the same mistake twice.`
```

			label vailheadhome
			`	Heading back to the spaceport after a day of hoverboarding, Taely surprises you by pulling the hood open on the hovercar and spending almost an hour going over the internal systems before closing it up with a nod. "Well, that was interesting. The hovercar, I mean." She gestures at it with her commlink. "So many different manufacturers were involved in making this vehicle. Sometimes I wonder if humanity appreciates the luxury such diversity affords them." She looks thoughtful, then gestures toward the ship. "Well, <first>, I promised you a tour of a historic Remnant district. Caelian has perhaps the best example of one, so I would recommend we go there."`
	on complete
		conversation

> 			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with gulls and pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a a few minutes they deposit you on a large platform.`

Reviewer 3: ```suggestion
			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with Gulls and Pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a a few minutes they deposit you on a large platform.`
```

> 			`	Approaching the edge of the platform, Taely puts out her arm to stop you from advancing any farther. "Watch out for the barrier." She says as she puts a hand out and touches something in mid-air in front of you. As she leans on it you can see faint ripples around her hand. "They are an essential safety feature and will not cause damage, but they can still hurt if you run into them."`

Reviewer 3: ```suggestion
			`	Approaching the edge of the platform, Taely puts out her arm to stop you from advancing any farther. "Watch out for the barrier," she says as she puts a hand out and touches something in mid-air in front of you. As she leans on it you can see faint ripples around her hand. "They are an essential safety feature, but they can still hurt if you run into them."`
```

> 			`	Moments later you see why the barriers are in place, as a large train whistles into the station at high speed and slows quickly to a stop in front of you. The barrier vanishes, and Taely leads you onto the train. Looking up and down the length of it, you can see hundreds, maybe even thousands of Remnant entering or exiting the train. Within moments the doors close and the train starts moving again and you can feel yourself being pressed back into the seat. You see a few lights flicker past outside the window, but soon there is nothing outside but a monotonous dark blur.`

Reviewer 3: ```suggestion
			`	Moments later, you see why the barriers are in place as a large train whistles into the station at high speed before stopping. The barrier vanishes, and Taely leads you onto the train. Looking up and down the length of it, you can see hundreds, maybe even thousands of Remnant entering and exiting the train. Within moments the doors close and the train starts moving again as you are pressed back into your seat. You see a few lights flicker past outside the window, but soon there is nothing outside but a monotonous dark blur.`
```

> 			`	Taely notices your gaze. "Sorry, but there will not be much to see out there. The entire train line is quite far underground. Far enough to simply pass under terrain obstacles like rivers, valleys, or chasms." She nods to herself in satisfaction. "This will not be a long trip: We will only spend a few minutes at top speed before we start decelerating to our destination. I would recommend buckling into the harness as the deceleration will be significant."`

Reviewer 3: ```suggestion
			`	Taely notices your gaze. "Sorry, but there will not be much to see out there. The entire train line is quite far underground. Far enough to simply pass under terrain obstacles like rivers, valleys, or chasms." She nods to herself in satisfaction. "This will not be a long trip: we will only spend a few minutes at top speed before we start decelerating near our destination. I would recommend buckling into the harness."`
```

		conversation
			`	You are just settling down at a table in the cafeteria when Taely slides onto the bench opposite you. "Greetings, <first>. How is life as an adventurer?"`
			choice
				`	"Exciting as always."`
					goto paradiseexciting
				`	"I am taking a break right now."`
			`	She nods sympathetically. "Things have been a bit hectic lately, what with dimensional pockets and mysterious new aliens. I can understand wanting to slow down a bit. With the launch of two new ships in such a short time, I know I feel like I need to just relax a bit." She sighs. "It makes me feel guilty, honestly. I have accomplished so much, and there are so many things to be grateful for, but I even have more projects in development to focus on." She looks lost in thought for a moment.`
				goto paradisechoice
			label paradiseexciting
			`	"I can only imagine what it must be like, constantly meeting new people and traveling to new places." She eats mechanically, lost in thought for a moment.`
			label paradisechoice
			`	Taely looks up at you. "Could I ask you something about life back in ancestral space?"`
			choice
				`	"Sure."`
					goto askquestion
				`	"Not right now."`
			label vaillater

> 			`	"Oh, maybe some other time, then." She looks disappointed, and settles in to eat."`

Reviewer 4: ```suggestion
			`	"Oh, maybe some other time, then." She looks disappointed, and settles in to eat.`
```
Extra quotation mark?

			label vailheadhome
			`	Heading back to the spaceport after a day of hoverboarding, Taely surprises you by pulling the hood open on the hovercar and spending almost an hour going over the internal systems before closing it up with a nod. "Well, that was interesting. The hovercar, I mean." She gestures at it with her commlink. "So many different manufacturers were involved in making this vehicle. Sometimes I wonder if humanity appreciates the luxury such diversity affords them." She looks thoughtful, then gestures toward the ship. "Well, <first>, I promised you a tour of a historic Remnant district. Caelian has perhaps the best example of one, so I would recommend we go there."`
	on complete
		conversation

> 			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with Gulls and Pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a a few minutes they deposit you on a large platform.`

Reviewer 4: ```suggestion
			`Taely directs you to a landing pad on the outskirts of the main spaceport, in an area that seems mostly filled with Gulls and Pelicans shifting larger quantities of products. She leads you down a large series of escalators that branch off in several directions, occasionally connecting with large ramps filled with steady stream of cargo traffic. After a few minutes they deposit you on a large platform.`
```
